### PR TITLE
Update style to include the summary cards for groups and groups_users

### PR DESCRIPTION
**Pull request by T-Dnzt**
[Related Issue](https://github.com/greencommons/commons/issues/79)

This pull request adds some examples of the summary cards usage to the `/style` page. I included the [highlight.js](https://highlightjs.org) library directly in the style view because we don't need it anywhere else. Also, there is no support for slim apparently so I'm not sure if we want to keep it or not... 

To have some data to show, I used the factories from the project. I don't think we want to mess with the DB but we could maybe use the first group if any and fallback on the factory (which has some disadvantages and might fail in the future since we don't have a persisted object with an `id` for example).

Here are screenshots of the addition:

screen shot 2559-12-14 at 5 04 05 pm
screen shot 2559-12-14 at 5 04 09 pm

### CODE REVIEW

File: app/views/pages/style.html.erb [hunk 1]
```diff
@@ -1,9 +1,90 @@
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/default.min.css">
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>
+<script>hljs.initHighlightingOnLoad();</script>
+
 <div class="text-center">
```
Comment (ptrikutam) [inline app/views/pages/style.html.erb]: Small nit -- can you place this below the additions you've made? I want to keep the original theme styles and our custom components separate.

**Comment by ptrikutam** [comment]
Hey @T-Dnzt -- Syntax highlighting isn't required. Just a `<pre>` block should be enough, I think. Highlighting support was the motivation to keep both `.html.erb` and `.slim`, we can get rid of the `.html.erb` files.

File: app/views/pages/style.html.erb (hunk 1 above)
Comment (T-Dnzt) [inline app/views/pages/style.html.erb]: Oh certainly, I will change it.

**Comment by T-Dnzt** [comment]
@ptrikutam Just removed the `.erb` version, we don't need it anymore. I also removed the highlighting and created a separate section title "Green Commons Visual Components" so people can easily understand what's part of the theme and what's part of Green Commons.

**Pull request merged**